tests/ui/const-generics/issues/issue-100313.stderr

```diff
@@ -2,7 +2,7 @@ error[E0080]: writing to ALLOC0 which is read-only
   --> $DIR/issue-100313.rs:18:5
    |
 LL |     x.set_false();
-   |     ^^^^^^^^^^^^^ evaluation of constant value failed here
+   |     ^^^^^^^^^^^^^ evaluation of `T::<&true>::set_false` failed here
```

Comment:
Why does this use the name of the function rather than the name of the surrounding constant...?